jOOQ-codegen/src/main/java/org/jooq/codegen/DefaultGeneratorStrategy.java

```diff
@@ -18,36 +18,14 @@
  * database integrations.
  *
  * For more information, please visit: http://www.jooq.org/licenses
- *
- *
- *
- *
- *
- *
- *
- *
- *
- *
- *
- *
- *
- *
- *
- *
```

Comment:
Please don't change these things. They're here for good reasons: https://github.com/jOOQ/jOOQ/blob/master/CONTRIBUTING.md